### PR TITLE
Changed content of Select Application Type and SPA pages

### PR DESCRIPTION
**Pull request by minolih**

### CODE REVIEW

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/applications/components/wizard/oauth-protocol-settings-wizard-form.tsx [hunk 1]
```diff
@@ -495,10 +495,8 @@ export const OauthProtocolSettingsWizardForm: FunctionComponent<OAuthProtocolSet
                                                     inboundOIDC.fieldscallBackUrls.info"
                                                     tOptions={ { callBackURLFromTemplate: callBackURLFromTemplate  } }
                                                     >
-                                                        This field is required for a functional app. 
-                                                        However, if you are planning to 
-                                                        try out the sample app, 
-                                                        you can use <strong>{ callBackURLFromTemplate }</strong>.
+                                                        Don’t have an app? Try out the sample app
```
Comment (ayshsandu) [inline apps/console/src/features/applications/components/wizard/oauth-protocol-settings-wizard-form.tsx:498]: `...Try out the sample app...` -> `... Try out a sample app ....`

File: apps/console/src/features/applications/data/application-templates/templates/android-mobile-application/create-wizard-help.tsx [hunk 2]
```diff
@@ -42,16 +42,16 @@ const AndroidApplicationCreateWizardHelp: FunctionComponent<AndroidApplicationCr
...
+            <p>A unique name to identify your application.</p>
             <p>E.g., My App</p>
 
             <Divider />
 
             <React.Fragment>
                 <Heading as="h5">Authorized redirect URIs</Heading>
                 <p>
-                    The redirect URL determines where the authorization code is sent to
+                    The redirect URLs determine where the authorization code is sent to
```
Comment (ayshsandu) [inline apps/console/src/features/applications/data/application-templates/templates/android-mobile-application/create-wizard-help.tsx:54]: `The redirect URLs determine` - > `The authorized redirect URLs determine`

File: apps/console/src/features/applications/data/application-templates/templates/oidc-web-application/create-wizard-help.tsx [hunk 3]
```diff
@@ -59,7 +59,7 @@ const OIDCWebApplicationCreateWizardHelp: FunctionComponent<OIDCWebApplicationCr
             <React.Fragment>
                 <Heading as="h5">Authorized URIs</Heading>
                 <p>
-                    The authorized URI determines where the authorization code is sent to
+                    The authorized URIs determine where the authorization code is sent to
```
Comment (ayshsandu) [inline apps/console/src/features/applications/data/application-templates/templates/oidc-web-application/create-wizard-help.tsx:62]: The redirect URLs determine - > The authorized redirect URLs determine

Comment (ayshsandu) [reply apps/console/src/features/applications/data/application-templates/templates/oidc-web-application/create-wizard-help.tsx:62]: Check other places too

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 4]
```diff
@@ -1141,8 +1141,8 @@ export const console: ConsoleNS = {
                                         "However, if you are planning to try the sample app, " +
                                         "this field can be ignored."
                                 },
-                                info: "This field is required for a functional app. However, if you are"  +
-                                "planning to try out the sample app, you can use {{callBackURLFromTemplate}}."
+                                info: "Don’t have an app? Try out the sample app using {{callBackURLFromTemplate}} " +
```
Comment (ayshsandu) [inline modules/i18n/src/translations/en-US/portals/console.ts:1144]: Try out a sample

File: apps/console/src/features/applications/components/wizard/oauth-protocol-settings-wizard-form.tsx (hunk 1 above)
Comment (minolih) [reply apps/console/src/features/applications/components/wizard/oauth-protocol-settings-wizard-form.tsx]: Fixed

File: apps/console/src/features/applications/data/application-templates/templates/android-mobile-application/create-wizard-help.tsx (hunk 2 above)
Comment (minolih) [reply apps/console/src/features/applications/data/application-templates/templates/android-mobile-application/create-wizard-help.tsx]: Fixed in new commit

File: apps/console/src/features/applications/data/application-templates/templates/oidc-web-application/create-wizard-help.tsx (hunk 3 above)
Comment (minolih) [reply apps/console/src/features/applications/data/application-templates/templates/oidc-web-application/create-wizard-help.tsx]: Fixed in new commit

File: modules/i18n/src/translations/en-US/portals/console.ts (hunk 4 above)
Comment (minolih) [reply modules/i18n/src/translations/en-US/portals/console.ts]: Fixed in new commit

**Review by brionmario — APPROVED** [review]

**Pull request merged**